tests/cypress/tests/e2e/admin-dashboard/admin-dashboard.stepdef.js

```diff
@@ -0,0 +1,25 @@
+import { When, Then } from "@badeball/cypress-cucumber-preprocessor";
+
+When("I select a state", () => {
+  cy.get("#state").select("MN"); // states dropdown;
+});
+
+When("I click on the {string} button", () => {
```

Comment:
**Suggestion**

I think this could reuse the `the form is submitted` step definition from `support/step_definitions/formInputs.js`

If not, the `{string}` should be used as a parameter so you don't need to hard code `Go to Report Dashboard`